peachjam_api/views.py

```diff
@@ -29,6 +31,11 @@ class RelationshipViewSet(viewsets.ModelViewSet):
     serializer_class = RelationshipSerializer
 
 
+class AnnotationViewSet(viewsets.ModelViewSet):
```

Comment:
this will need to be restricted to the user making the api call.

Reply:
done